.github/workflows/CI-Fast-Tooling

```diff
@@ -0,0 +1,28 @@
+name: CI-Fast-Tooling
+
+on:
+  push:
+    branches: [ release ]
```

Comment:
Reviewing https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-multiple-events-with-activity-types-or-configuration looks like the syntax for specifying branches should be:

```
branches:
  - release
```